content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -91,6 +91,14 @@ For example, the line below states that the task must be started every Friday at
 
 To generate CronJob schedule expressions, you can also use web tools like [crontab.guru](https://crontab.guru/).
 
+## TimeZone Support
+{{< feature-state for_k8s_version="v1.24" state="alpha" >}}
+
+There is now a new `timeZone` field and requires the `CronJobTimeZone` featuregate to be enabled to use. It is an optional field that should be set either to nil or a non-empty string. The `timeZone` string value should be a valid time zone name. Please see: https://en.wikipedia.org/wiki/List_of_tz_database_time_zones. If not specified, the local time zone of kube-controller-manager is used.
+
+A time zone database from the Go standard library is included in the binaries and used as a fallback in case an external database is not available.
```

Comment:
```suggestion
A time zone database from the Go standard library is included in the binaries and used as a fallback in case an external database is not available on the system. 
```